### DISCUSSION TITLE
Devops vs backend

### DISCUSSION THREAD

**Discussion by shanu13** [commclassroom/roadmaps#408] [Q&A]
Hey ,
I'm confused between devops and backend . I have done some backend projects with node.js , but now I did'nt getting interest with node.js
plesae suggest me which one I should go for having better remote jobs 
Devops or Backend with java

**Comment by ptiwari6239** [comment]
hi,
 Pursuing a career in DevOps can provide exciting opportunities to work with cutting-edge technologies, solve complex problems, and make a real impact on software development and delivery and also DevOps is a rapidly growing and in-demand field that offers a range of benefits, there are lots of backend developers in the market but very few good DevOps engineer
